### PR TITLE
Capstone project for the creation of my website.

### PR DESCRIPTION
**Pull request by ABDULALI3468**
In this project, I have created the website for my capstone which is a tourism website, which attracts tourists and we are it's for giving tourists the facilities they want. Moreover, I have added our staff members dynamically and a button feature also.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+  min-height: 500px;
+  padding: 140px 0 10%;
+}
+
+.front-page__h2 {
+  color: var(--Orange);
+  font-family: var(--fontFamily-lato);
+  font-weight: normal;
+  font-size: var(--fontSize-h2);
+}
```
Comment (medaminedev66) [inline style.css]: Let's stick to the original design as much as we can:
- [ ] I recommend using the `Cocogoose` font for the main title so your design matches the original design.
- [ ] In the mobile version, The main title of the home page should be aligned to the left. I recommend doing that so your design matches the original design.

- Your design:
image

- Original design:
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+  -webkit-text-fill-color: transparent;
+  -webkit-font-smoothing: antialiased;
+}
+
+.front-page__message {
+  border: 1px solid white;
+  padding: 2%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
```
Comment (medaminedev66) [inline style.css]: - [ ] Let's use the same background color that is provided in the original design for the message to improve your UI design and also to improve the contrast ratio of your page.
- [ ] I recommend using the `Lato` font for the text message. Remember that only two fonts should be used in your pages: `Lato` and `Cocogoose`.
- Your design:
image


- Original design: 
image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+  }
+
+  .features__cards {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    width: auto;
+    height: auto;
+    transition: border 1.4s;
+  }
```
Comment (medaminedev66) [inline style.css:566]: - [ ] The cards should have the same width, let's try using the same width for all the cards to improve the UI design.
- [ ] You need to hide the button `Tour with us` in the desktop version and add a link `See the whole ...` as provided in the original design.
- Your design:
image

File: about.css [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+  margin-right: 35px;
+}
+
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 250px;
+  background-color: rgba(242, 242, 242, 0.2) !important;
+}
```
Comment (medaminedev66) [inline about.css]: - [ ] On about page, the footer should be dark. Kindly work on that that so your design matches the original design.
- Your design:
image

- Original design:

image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,578 @@
...
+.tour_button {
+  height: 80px;
+  width: 80%;
+  margin: auto;
+  font-size: var(--fontSize-h3);
+  margin-top: 50px;
+  font-weight: bold;
+  color: var(--White);
+  background-color: var(--Orange);
+}
```
Comment (medaminedev66) [inline style.css]: - [ ] Let's remove the border from the tour button to improve your design.

image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        </button>
+        <nav class="navbar-main">
+          <div class="navbar-main-container">
+            <div class="navbar-main__nav">
+              <button class="navbar-button__close">x</button>
+              <a class="navbar-main__brand" href="#"
+                ><img
+                  src="./images/weblogo.png"
+                  alt="EME Olympiad Logo"
+              /></a>
```
Comment (medaminedev66) [inline about.html]: - [ ] On the about page, the menu should cover the whole mobile screen. Kindly work on that. 
image

[OPTIONAL] I recommend stopping the page from scrolling after the menu is open to improve the user experience.

File: README.md [hunk 7]
```diff
@@ -6,8 +6,6 @@
 
 > This project is for completing my Capstone Project requirements for first module at Microverse. It consists of two webpages for a touring company called Alpha Tours, which offers international tour packages.
 
-![screenshot]()
-
 It consists of two webpages Home and About with both being responsive to desktop and mobile layouts.
 
 ## Built With
```
Comment (medaminedev66) [inline README.md:12]: - [ ] The demo link is broken. Kindly use GitHub Pages to deploy the web pages and add the link to the deployed project to your README.

File: README.md [hunk 8]
```diff
@@ -54,4 +53,5 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```
Comment (medaminedev66) [inline README.md:57]: - [ ] The MIT link is broken, Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ABDULALI3468  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- Git/GitHub flow best practices have been followed ✔️
- There are no linter errors ✔️
- Your code organization was really good

## Required Changes ♻️
- [ ] Following the project requirements, you should record a video presenting the features of the project. Make sure that:
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

- [ ] The pull request description should be more descriptive. Kindly update that.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — CHANGES_REQUESTED** [review]
.

**Review by topeogunleye — APPROVED** [review]
# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
congratulations

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

> 
> ## [Optional] suggestions:
> * Please next time, add your 5 mins video link to the PR description as stated in the project requirement
>Screenshot from 2022-04-16 15-54-10

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**